test-suite/src/function/current_datetime.rs

```diff
@@ -0,0 +1,59 @@
+use {
+    crate::*,
+    gluesql_core::prelude::{Payload, Value::*},
+};
+
+test_case!(current_datetime, {
```

Comment:
ref. https://gluesql.org/docs/0.16.0/sql-syntax/functions/text/left

We aim to generate docs from the test-suite here.
This test itself looks good enough. but three different functions exist in a single test case `current_datetime`.
Then, it could be hard to make the automation flow.
Can we split this to have three test cases - `current_time`, `current_timestamp` and `current_date`.